data.sql

```diff
@@ -1,5 +1,15 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES 
+(1, 'Agumon', '2020-02-03', 0, 'TRUE', 10.23),
+(2, 'Gabumon', '2018-11-15', 2, 'TRUE', 8),
+(3, 'Pikachu', '2021-01-07', 1, 'FALSE', 15.04),
+(4, 'Devimon', '2017-05-12', 5, 'TRUE', 11),
+(5, 'Charmander', '2020-02-08', 0, 'FALSE', 11),
+(6, 'Plantmon', '2021-11-15', 2, 'TRUE', -5.7),
+(7, 'Squirtle', '1993-04-02', 3, 'FALSE', -12.13),
+(8, 'Angemon', '2005-06-12', 1, 'TRUE', -45),
+(9, 'Boarmon', '2005-06-07', 7, 'TRUE', 20),
```

Comment:
- [ ] The same here,  The weight_kg of the fifth data has a decimal value. Please add this 👍🏻Your code might look like this:

```sql

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Boarmon', '2005/06/07', '7', 'true', '20.4');